.github/ISSUE_TEMPLATE/documentation_report.md

```diff
@@ -1,61 +1,36 @@
 ---
 name: 📝 Documentation Report
 about: Ask us about docs
-
 ---
-
-<!---
-Verify first that your issue/request is not already reported on GitHub.
-THIS FORM WILL BE READ BY A MACHINE, COMPLETE ALL SECTIONS AS DESCRIBED.
-Also test if the latest release, and devel branch are affected too.
-ALWAYS add information AFTER (OUTSIDE) these html comments.
-Otherwise it may end up being automatically closed by our bot. -->
+<!--- Verify first that your improvement is not already reported on GitHub -->
+<!--- Also test if the latest release and devel branch are affected too -->
+<!--- This form is processed by Ansibot (machine), complete *all* sections as described -->
 
 ##### SUMMARY
-<!--- Explain the problem briefly -->
+<!--- Explain the problem briefly below -->
 
 ##### ISSUE TYPE
- - Documentation Report
+- Documentation Report
 
 ##### COMPONENT NAME
-<!--- Insert, BELOW THIS COMMENT, the name of the module, plugin, task or feature.
-Do not include extra details here, e.g. "vyos_command" not "the network module vyos_command" or the full path-->
+<!--- Write the short name of the module, plugin, task or feature below -->
 
 ##### ANSIBLE VERSION
-<!--- Paste, BELOW THIS COMMENT, verbatim output from "ansible --version" between quotes below -->
-```
+<!--- Paste verbatim output from "ansible --version" between quotes -->
+```paste below
 
 ```
 
 ##### CONFIGURATION
```

Comment:
This is optional, does it make sense for Docs issues?

Reply:
@acozine asked the same question. To me requests for documentation can also be a result from changes to your environment, the specific platform you use, etc. So in most situations it is probably not relevant, but it could be in some. I don't mind removing it, but I'd rather challenge the contributor in advance, than maybe never challenging him in the process itself.

Reply:
This could be helpful if the docs issue is related to use of the `ansible-doc` command line tool. I would say leave it because it provides info about Python version and config as well, and could give hints about someone trying to use a feature that is new, deprecated, or changed.

Reply:
Oh, woops. I misread this and thought it was talking about removing the `ANSIBLE VERSION` section. OS and STEPS TO REPRODUCE are ok to remove or just leave as optional. More info is always better.

Reply:
**STEPS TO REPRODUCE** was already removed. the **OS / ENVIRONMENT** section may still be needed though.